content/09.overfitting.md

```diff
@@ -19,3 +19,5 @@ If the model performs no better on real data than randomly scrambled data, then
 Additionally, one must be sure that their data are not skewed or biased, such as by having confounding and scientifically irrelevant variables that the model can pick up on [@arxiv:1807.00431].
 In this case, simply holding out test data is insufficient.
 The best remedy for confounding variables is to [know your data](#know-your-problem) and to test your model on truly independent data.
+
+In essence, split your data into training, tuning, and testing sets to assess the performance of the model on data it truly has not seen before, while also being cognizant of the danger of skewed or biased data artificially inflating accuracy.
```

Comment:
I'd focus on not using the test set multiple times in the actionable summary.  That is a major problem in many manuscripts.